index.html

```diff
@@ -238,17 +293,23 @@ <h2>Percentage of Economically Disadvantaged Students</h2>
 
 		<!--create time slider-->
 		  <div class='session' id='sliderbar'>
-		  <h2>Year: <label id='active-year'>1997</label></h2>
+		  <h2>Year: <label id='active-year'></label></h2>
```

Comment:
Note that the year is deliberately blank here.  It gets set by runWhenLoadComplete(), to match the visual state of the slider.